packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -255,6 +256,7 @@ class CupertinoTextField extends StatefulWidget {
     this.expands = false,
     this.maxLength,
     this.maxLengthEnforced = true,
+    this.maxLengthEnforcement = MaxLengthEnforcement.enforced,
```

Comment:
Let's remove this parameter from cupertino text field and deprecate the `maxLengthEnforced` parameter.

Reply:
What if there are more behaviors introduced into the enum in the future? Bring in the enforcement again?

Reply:
And how do we control the composing if there is no enforcement? 🤔

Reply:
And how do we control the composing if there is no enforcement? 🤔

Reply:
Oh sorry you're right. I think we need to document what `.none` does here since it behaves differently from TextField.